.github/workflows/build-front-end-development.yml

```diff
@@ -0,0 +1,28 @@
+name: deploy-development-front-end-container
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - develop
+    paths:
+      - "frontend/src/**"
```

Comment:
This addition is good to have in the develop branch! On master there is none since it should all ways be updated since the version number increase is in the front-end.